tests/e2e/tests/test_redirects.py

```diff
@@ -235,21 +235,21 @@ def test_stub_installer_redirect_for_en_us_and_win(self, base_url, product):
 
         parsed_url = urlparse(response.url)
 
-        assert (requests.codes.ok == response.status_code,
-                'Redirect failed with HTTP status. %s' %
-                self.response_info_failure_message(base_url, param, response))
+        assert requests.codes.ok == response.status_code, \
+            'Redirect failed with HTTP status. %s' % \
+            self.response_info_failure_message(base_url, param, response)
 
-        assert ('https' == parsed_url.scheme,
-                'Failed to redirect to the correct scheme. %s' %
-                self.response_info_failure_message(base_url, param, response))
+        assert 'https' == parsed_url.scheme, \
+            'Failed to redirect to the correct scheme. %s' % \
+            self.response_info_failure_message(base_url, param, response)
 
-        assert ('download-installer.cdn.mozilla.net' == parsed_url.netloc,
-                'Failed by redirected to incorrect host. %s' %
-                self.response_info_failure_message(base_url, param, response))
+        assert 'download-installer.cdn.mozilla.net' == parsed_url.netloc, \
+            'Failed by redirected to incorrect host. %s' % \
+            self.response_info_failure_message(base_url, param, response)
 
-        assert ('Unknown' != self.get_x_backend_server(response),
-                'Failed, x-backend-server was not in the response object %s' %
-                self.response_info_failure_message(base_url, param, response))
+        assert 'Unknown' != self.get_x_backend_server(response), \
```

Comment:
We can remove this assert, `x-backend-server` is no longer something that is in the response object (thx @oremj for weighing in on this).